### PR TITLE
[Workers] Adds new flag for wrangler commands

### PR DESCRIPTION
**Pull request by KaiSpencer**
A new flag for `wrangler dev` and `wrangler pages dev` is to be added in https://github.com/cloudflare/workers-sdk/pull/5099

This change covers the documentation for this new flag.

NB: the optional meta is ordered differently for each of these, I have stayed consistent with what is currently in place for the other flags in the respective sections.

### CODE REVIEW

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -691,6 +691,8 @@ As of Wrangler v3.2.0, `wrangler dev` is supported by any Linux distributions pr
   - Exposes a `/__scheduled` fetch route which will trigger a scheduled event (Cron Trigger) for testing during development. To simulate different cron patterns, a `cron` query parameter can be passed in: `/__scheduled?cron=*+*+*+*+*`.
 - `--log-level` {{<type>}}"debug"|"info"|"log"|"warn"|"error"|"none"{{</type>}} {{<prop-meta>}}(default: log){{</prop-meta>}} {{<prop-meta>}}optional{{</prop-meta>}}
   - Specify Wrangler's logging level.
+- `--show-interactive-dev-session` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: true){{</prop-meta>}} {{<prop-meta>}}optional{{</prop-meta>}}
```
Comment (petebacondarwin) [inline content/workers/wrangler/commands.md]: Strictly the default is to go with whether the stdin has a TTY or not.

Comment (petebacondarwin) [inline content/workers/wrangler/commands.md]:
```suggestion
- `--show-interactive-dev-session` {{<type>}}boolean{{</type>}} {{<prop-meta>}}(default: true if the terminal supports interactivity){{</prop-meta>}} {{<prop-meta>}}optional{{</prop-meta>}}
```

File: content/workers/wrangler/commands.md [hunk 2]
```diff
@@ -1699,6 +1701,8 @@ wrangler pages dev [<DIRECTORY>] [OPTIONS] [-- <COMMAND...>]
   - Runtime compatibility flags to apply.
 - `--compatibility-date` {{<type>}}string{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
   - Runtime compatibility date to apply.
+- `--show-interactive-dev-session` {{<type>}}boolean{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}} {{<prop-meta>}}(default: true){{</prop-meta>}}
```
Comment (petebacondarwin) [inline content/workers/wrangler/commands.md]:
```suggestion
- `--show-interactive-dev-session` {{<type>}}boolean{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}} {{<prop-meta>}}(default: true if the terminal supports interactivity){{</prop-meta>}} 
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

**Review by petebacondarwin — APPROVED** [review]
Approving subject to the suggested changes.

**Review by kodster28 — APPROVED** [review]

**Pull request merged**